lottery/Cargo.toml

```diff
@@ -10,7 +10,7 @@ gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"] }
 codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive", "full"] }
 scale-info = { version = "2.0.1", default-features = false, features = ["derive"] }
 primitive-types = { version = "0.11.1", default-features = false, features = ["scale-info"] }
-sp-core-hashing = { version = "4.0.0", default-features = false }
+sp-core = { version = "6.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", default-features = false, features = ["full_crypto"] }
```

Comment:
Why do you use heavier `sp-core` instead of lighter `sp-core-hashing`?

Reply:
Fixed